objectivec/GPBMessage.m

```diff
@@ -3086,7 +3086,12 @@ + (BOOL)resolveInstanceMethod:(SEL)sel {
   // no need for checks here to see if has*/setHas* are allowed.
 
   ResolveIvarAccessorMethodResult result = {NULL, NULL};
-  for (GPBFieldDescriptor *field in descriptor->fields_) {
+
+  
+  // |field| can be __unsafe_unretained because they are created at startup
+  // and are essentially global. No need to pay for retain/release when
+  // they are captured in blocks.
```

Comment:
Can you dup this comment on the two other places that added `__unsafe_unretained`.  Or maybe a block comment before the two statics explain the use generally in this part of the code?

Reply:
Done